[2023-03-20 14:36:04,488] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dag_with_catchup_backfill_v01.task1 manual__2023-03-20T05:36:02.344258+00:00 [queued]>
[2023-03-20 14:36:04,490] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dag_with_catchup_backfill_v01.task1 manual__2023-03-20T05:36:02.344258+00:00 [queued]>
[2023-03-20 14:36:04,490] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-03-20 14:36:04,491] {taskinstance.py:1239} INFO - Starting attempt 1 of 4
[2023-03-20 14:36:04,491] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-03-20 14:36:04,493] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): task1> on 2023-03-20 05:36:02.344258+00:00
[2023-03-20 14:36:04,495] {standard_task_runner.py:52} INFO - Started process 15475 to run task
[2023-03-20 14:36:04,497] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_catchup_backfill_v01', 'task1', 'manual__2023-03-20T05:36:02.344258+00:00', '--job-id', '81', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_catchup_and_backfill.py', '--cfg-path', '/var/folders/x1/7qlvjnbj06q1nkyhy70vbb2c0000gn/T/tmputee5d4s', '--error-file', '/var/folders/x1/7qlvjnbj06q1nkyhy70vbb2c0000gn/T/tmpprtetjh9']
[2023-03-20 14:36:04,498] {standard_task_runner.py:77} INFO - Job 81: Subtask task1
[2023-03-20 14:36:04,514] {logging_mixin.py:109} INFO - Running <TaskInstance: dag_with_catchup_backfill_v01.task1 manual__2023-03-20T05:36:02.344258+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2023-03-20 14:36:04,528] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=leebyeonghwa
AIRFLOW_CTX_DAG_ID=dag_with_catchup_backfill_v01
AIRFLOW_CTX_TASK_ID=task1
AIRFLOW_CTX_EXECUTION_DATE=2023-03-20T05:36:02.344258+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-20T05:36:02.344258+00:00
[2023-03-20 14:36:04,528] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/x1/7qlvjnbj06q1nkyhy70vbb2c0000gn/T
[2023-03-20 14:36:04,528] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo This is a simple bash command!']
[2023-03-20 14:36:04,531] {subprocess.py:85} INFO - Output:
[2023-03-20 14:36:04,533] {subprocess.py:89} INFO - This is a simple bash command!
[2023-03-20 14:36:04,533] {subprocess.py:93} INFO - Command exited with return code 0
[2023-03-20 14:36:04,538] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=dag_with_catchup_backfill_v01, task_id=task1, execution_date=20230320T053602, start_date=20230320T053604, end_date=20230320T053604
[2023-03-20 14:36:04,559] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-03-20 14:36:04,565] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
